server/{{appName}}-app/src/main/genesis/cfg/{{appName}}-processes.xml

```diff
@@ -0,0 +1,93 @@
+<!--
+!
+!   System              : Genesis Business Library
+!   Sub-System          : multi-pro-code-test Configuration
+!   Version             : 1.0
+!   Copyright           : (c) Genesis
+!   Date                : 2022-03-18
+!   Function : Provide Process Definitions Config for multi-pro-code-test.
+!
+!   Modification History
+!
+-->
+<processes>
+<!--    <process name="{{upperCase appName}}_DATASERVER">-->
```

Comment:
This is intentionally commented out?

Reply:
Yes, these are here to help the user get started. They may not apply to all projects so they are commented out.